docs/references/bib.bib

```diff
@@ -1372,3 +1372,16 @@ @mastersthesis{holloway:1988a
 	url = {http://ir.library.oregonstate.edu/xmlui/bitstream/handle/1957/39942/HollowayJamesL1989.pdf},
 	year = {1988}
 }
+
+@article{baudin:2012,
```

Comment:
Alas, not yet. :( My typical workflow is to grab a template (e.g., an article template from https://github.com/stdlib-js/stdlib/tree/develop/docs/references/templates). Copy to the database. And then fill in the various items. Some sites allow you to export a `*.bib` file, but I usually only use this as a reference, as everyone seems to have their own style for `*.bib` formatting. 

We have automated, to some degree, inserting links into the link database `make links-insert`, but have yet to create similar tooling for references.

Until that point, by hand is how we do it.